pkg/pillar/types/clustertypes.go

```diff
@@ -80,6 +80,8 @@ type EdgeNodeClusterStatus struct {
 	// This node needs to be up first before other nodes can join the cluster. This BootstrapNode
 	// will own the 'JoinServerIP' on it's cluster interface.
 	BootstrapNode bool
+	// EncryptedClusterToken - for kubernetes cluster server token
```

Comment:
Can you specify how this is encrypted in the comment?

Reply:
Sure, updated with more comments